### DISCUSSION TITLE
InspireHep BibTex export fields

### DISCUSSION THREAD

**Discussion by jorgetc16** [retorquere/zotero-better-bibtex#2418] [Q&A]
Hi,
Is there a way that when exporting with BBT you can get the same fields as when exporting from Inspire?

An exemple of what I get with Inspire:
```
@article{Castillo:2022zfl,
    author = "Castillo, Andr\'es and Martin-Camalich, Jorge and Terol-Calvo, Jorge and Blas, Diego and Caputo, Andrea and Santos, Ricardo Tanaus\'u G\'enova and Sberna, Laura and Peel, Michael and Rubi\~no-Mart\'\i{}n, Jose Alberto",
    title = "{Searching for dark-matter waves with PPTA and QUIJOTE pulsar polarimetry}",
    eprint = "2201.03422",
    archivePrefix = "arXiv",
    primaryClass = "astro-ph.CO",
    doi = "10.1088/1475-7516/2022/06/014",
    journal = "JCAP",
    volume = "06",
    number = "06",
    pages = "014",
    year = "2022"
}
```

vs what I get with Zotero+BBT:
```
@article{Castillo:2022zfl,
  title = {Searching for Dark-Matter Waves with {{PPTA}} and {{QUIJOTE}} Pulsar Polarimetry},
  author = {Castillo, Andrés and Martin-Camalich, Jorge and Terol-Calvo, Jorge and Blas, Diego and Caputo, Andrea and Santos, Ricardo Tanausú Génova and Sberna, Laura and Peel, Michael and Rubiño-Martín, Jose Alberto},
  date = {2022-06-13},
  journaltitle = {Journal of Cosmology and Astroparticle Physics},
  shortjournal = {JCAP},
  volume = {06},
  number = {06},
  eprint = {2201.03422},
  eprinttype = {arxiv},
  primaryclass = {astro-ph.CO},
  pages = {014},
  issn = {1475-7516},
  doi = {10.1088/1475-7516/2022/06/014},
  url = {http://arxiv.org/abs/2201.03422},
  urldate = {2023-02-17},
  archiveprefix = {arXiv},
  keywords = {Astrophysics - Cosmology and Nongalactic Astrophysics,High Energy Physics - Phenomenology},
  annotation = {5 citations (INSPIRE 2023/2/17) 5 citations w/o self (INSPIRE 2023/2/17)},
  file = {/home/jorgetc/Zotero/storage/A3237WNT/Castillo et al. - 2022 - Searching for dark-matter waves with PPTA and QUIJ.pdf;/home/jorgetc/Zotero/storage/KVY2TZCV/2201.html}
}
```
The former one is simpler plus I have problems with journal vs journaltitle/shortjournal, journal works and appears in my article when compiling but journaltitle or shortjournal do not.

**Comment by retorquere** [comment]
Can you right-click that entry in Zotero and send a debug log? There's a few fields in the 2nd entry that I don't expect to see in a BBT export.

**Comment by jorgetc16** [comment]
Here you have the debug log, should I post it in the issues page as well?
[YR8M57IW-refs-euc.zip](https://github.com/retorquere/zotero-better-bibtex/files/10804764/YR8M57IW-refs-euc.zip)

**Reply by retorquere** [reply]
No need unless we need to change BBT. The item in `YR8M57IW-refs-euc` contains substantially more info than the first entry above. You can shave that down to what is in the first entry during export by a combination of skipfields, checking "use journal abbreviation" during export, and/or a postscript.

**Reply by jorgetc16** [reply]
Ok, I think I got it. Thank you!

**Reply by xju2** [reply]
Hi @jorgetc16 , could you please share the formula for BIBTex generating citation key so the generated key is the same as the ones generated by InspireHEP?

**Reply by retorquere** [reply]
`inspireHep` will do that.
